### PR TITLE
Add the ToNumber preprocess layer.

### PR DESCRIPTION
**Pull request by brightcoder01**

### CODE REVIEW

**Review by terrytangyuan — COMMENTED** [review]
Is it really necessary to add this to elasticdl? The preprocessing layers seem so simple and I think they should be in examples instead.

**Comment by brightcoder01** [comment]
> Is it really necessary to add this to elasticdl? The preprocessing layers seem so simple and I think they should be in examples instead.

Good question.
The layers is useful for the code_gen of SQLFlow transform expression and can be imported directly in the generated code. If not, I'm afraid we should generate this piece of code multiple times.

**Comment by terrytangyuan** [comment]
@brightcoder01 Sure but maybe this can be moved to examples instead of a module/folder?

File: elasticdl_preprocessing/tests/to_number_test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import unittest
+
+import numpy as np
+import tensorflow as tf
+
+from elasticdl_preprocessing.layers.to_number import ToNumber
+
+
+def sparse_tensor_equal(sp_a, sp_b):
```
Comment (workingloong) [inline elasticdl_preprocessing/tests/to_number_test.py]: We may use the method in multiple unit tests of preprocessing layers. Can we move it to a single file like `tests/test_utils.py` ?

Comment (brightcoder01) [reply elasticdl_preprocessing/tests/to_number_test.py]: Good idea.

**Comment by brightcoder01** [comment]
> @brightcoder01 Sure but maybe this can be moved to examples instead of a module/folder?

Good suggestion. We will add a model example for all the preprocessing layers after submitting all the layers.

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**